packages/style/tsconfig.json

```diff
@@ -16,7 +15,6 @@
     "noImplicitAny": true,
     "baseUrl": ".",
     "allowSyntheticDefaultImports": true,
-    "jsx": "react-jsx",
```

Comment:
are you sure that's not needed?

Reply:
I assume it's not needed because we don't have react, do you think we should keep it anyway?

Reply:
I saw you removed the react dep entirely  that seems cool